Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/RoomWrapper.java

```diff
@@ -29,14 +30,16 @@
     private final List<PlayerWrapper> players = new ArrayList<>();
     private final Map<PlayerWrapper, Team> teams = new HashMap<>();
 
-    private final List<String> mutedNames = new ArrayList<>();
-
     private final SessionWrapper session;
     private final Room room;
 
+    private final Chat chat;
+
     public RoomWrapper(SessionWrapper session, Room room) {
         this.session = session;
         this.room = room;
+
+        chat = new Chat(this);
     }
 
     public void join(PlayerWrapper player, String password) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)